assets/app/styles/_core.less

```diff
@@ -112,9 +122,18 @@ html, body {
       float: left;
       width: @navbar-project-menu-sm;
     }
+    .navbar-filter-widget {
+      /* prevent the filter button from wrapping at wide
+         screen sizes */
+      min-width: 340px;
+    }
     .navbar-search {
       margin-left: @sidebar-left-width-open-sm;
     }
+    .create-button-container {
+      text-align: right;
+      margin: 30px 0;
```

Comment:
ok, maybe just a comment then, to help our future selves, e.g. "Top-margin to align with other items that have labels above them"